src/internal/packager2/layout/create.go

```diff
@@ -145,7 +145,7 @@ func CreatePackage(ctx context.Context, packagePath string, opt CreateOptions) (
 			ImageList:             componentImages,
 			Arch:                  pkg.Metadata.Architecture,
 			RegistryOverrides:     opt.RegistryOverrides,
-			CacheDirectory:        filepath.Join(cachePath, ImagesDir),
+			CacheDirectory:        filepath.Join(cachePath, zoci.ImageCacheDirectory),
```

Comment:
Thoughts on changing `images.PullConfig.CacheDirectory` to

```
type CacheConfig struct {
  Directory string
}
```

Users may want to access to varying cache patterns in the future (for example, skip cache) and this allows us to add keys without introducing breaking existing behaviors.